.docker/docker-compose-prod.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+services:
+	postgress-service:
+		image: postgres:15.1-alpine
+		container_name: postgress-db
+		restart: always
+		environment:
+			- POSTGRES_DB=organizer_db
+			- POSTGRES_PASSWORD=password
+		volumes: 
+			- postgres:/var/lib/postgres
+
+	organizer-service:
+		image: ghcr.io/felixsteinke/organzier-application-demo:latest
+		environment: 
+			- ConnectionStrings:PostgresDB=Host=postgres-service:5432;Database=organizer_db;Username=postgres;Password=password
```

Comment:
We need to check, if there are some `" "` required. Maybe you need to build it locally once, to figure it out.